docs/conf.py

```diff
@@ -278,11 +278,19 @@
 autodoc_default_flags = ['members']
 
 
-MOCK_MODULES = ['pycurl', 'wrapanapi', 'ovirt-engine-sdk-python',
+MOCK_MODULES = ['pycurl', 'wrapanapi', 'ovirt-engine-sdk-python', 'wrapanapi.utils',
```

Comment:
i believe we might want to move the growing listing of modules into a text file in future in order to make a full listing more nicely readable and editable